docs/config.rst

```diff
@@ -1,11 +1,507 @@
 Configuring CumulusCI
 =====================
 
-cumulusci.yml
--------------
 
-Global overrides
-----------------
 
-Local overrides
----------------
+``cumulusci.yml`` Structure
+---------------------------
+There are six top level sections of the ``cumulusci.yml`` file: Project, Tasks, Flows, Sources, Orgs, and Plans.
+
+* **Project** - This section is largely generated by running ``cci project init`` and has two major sub-categories: ``package``, and ``git``. The ``package`` sub-section contains information about the project's package, while the ``git`` sub-section contains a link to the project's GitHub repository and the branch prefixes being utilized.
```

Comment:
Also has `name` and `dependencies`

Reply:
I'm going to remove the specifics from this section and keep it more general. Am going to link to a cumulusci.yml reference section.